charts/catalog/values.yaml

```diff
@@ -67,11 +67,6 @@ apiserver:
     logPath: "/tmp/service-catalog-apiserver-audit.log"
   serviceAccount: service-catalog-apiserver
 controllerManager:
-  # controller-manager image to use
-  image: quay.io/kubernetes-service-catalog/controller-manager:v0.0.23
```

Comment:
I think there is still value in being able to specify each image separately. For example: canary apiserver with a specific known version of the controller-manager..

Reply:
My issue with this is that it then does not resolve the main crux of the issue, where the user unintentionally uses different versions of apiserver and controller-manager. Who do you see using separate versions and for what purpose? I am imagining that the target audience for that use case would be fairly advanced, perhaps advanced enough to modify the helm charts themselves manually.

Perhaps I could see adding images values for apiserver and controller-manager in addition to the global ones. So if the user sets the values for the specific component, then that would override the global ones. But I do not like the idea of requiring the user to set the value for both components individually in most scenarios.

Reply:
> For example: canary apiserver with a specific known version of the controller-manager..

@MHBauer just make a separate PR with only changes you want, no? I don't see any benefits in testing canary apiserver with release controller-manager, if you changed both in your branch.

Reply:
I'm fine with being able to override it, but I think the chart should use a single image unless you specifically override the api server or controller manager image.